config/initializers/doorkeeper.rb

```diff
@@ -163,7 +163,7 @@
   # For more information go to
   # https://github.com/doorkeeper-gem/doorkeeper/wiki/Using-Scopes
   #
-  default_scopes :public
+  default_scopes :public, :read_articles
```

Comment:
I'm not sure `read_articles` is intuitive, maybe just `articles`?

`public`, the default, will be applied to all public resources
`articles` garanties access to ALL articles

and so on...

Reply:
Presumably `read_articles` contrasts to `write_articles` in this case?

Would we maybe have `read_classified_listings` and `read_classified_listings`?

And then maybe `read_settings` and `write_settings` or something for managing the user's personal data?

Basically we could have `read` and `write` for every resource?

Reply:
It depends on how granular this thing has to be, what does public mean then if we have read and write for every resource?

Will it mean that eventually the oauth page could have 10 choices?

I'm not advocating against it in general, I'm not sure about what we want to accomplish with scopes probably

Reply:
I am not 100% satisfied with the name, it's actually more like "read_all_articles"(including drafts), not "read_articles".
But I suppose that "read" and "write" scopes should be separated.
"Public" means reading only public data.

Reply:
maybe scopes can be scoped, `articles:read` or `articles:write`. Resource and action seems to me more intuitive than the other way around but it's just a detail.

Reply:
If we want to generalize, a scope could be "read_drafts" (unpublished articles, listings, etc). I'm suspicious about putting read and write into the same scope. E.g. if a user only wants the 3-rd party to pull articles, why give permissions to update them?

Reply:
But I would rather go with many options (in the future) than generalizing.